vite.config.js

```diff
@@ -40,6 +40,10 @@ const config = {
           src: "./node_modules/bootstrap/dist/js/bootstrap.bundle.min.*",
           dest: "./assets/dist/vendor",
         },
+        {
+          src: "./node_modules/htmx.org/dist/htmx.js",
```

Comment:
Could we use the minified version here instead?

```suggestion
          src: "./node_modules/htmx.org/dist/htmx.min.js",
```